lib/season_statable.rb

```diff
@@ -0,0 +1,103 @@
+module SeasonStatable
+
+  def all_season_game_id(season)
+    @games.map do |game|
+      game.game_id if game.season == season
+    end.compact 
+  end 
+
+  def winningest_coach(season)
+    coach_wins = @game_teams.each_with_object(Hash.new([0,0])) do |game, hash|
```

Comment:
this and worst coach can definitely be parsed down into more helpers. Let's chat about that today as far as next steps.